week-9/Homework/mandatory/1-practice/2-code-reading.md

```diff
@@ -32,7 +34,9 @@ console.log(f1())
 console.log(y)
 ```
 
-What will be the output of this code. Explain your answer in 50 words or less.
+What will be the output of this code.// Explain your answer in 50 words or less.
+Line 30 will print 10 to the console because x is a global variable.
+Line 34 will print y is not defined, this is because y is a local variable only accessible inside the functiion. So this x it is out of scope for the console.log called on line 6. The function itself should return undefined.
```

Comment:
Correct! (I am not sure what you are referring to on line 6 however, I think you might have a typo)